.changeset/128-responsive-layout-init

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
When we discussed this, I said I don't think the file name matters. But I do wonder if it matters whether there's a `.md` suffix. The changesets CLI always creates files with that suffix, and I noticed the same for the suggested filename when I used the GitHub flow in #135.

Reply:
renamed it.